# This is an action definition file, which has three parts: 
# the goal, the result, and the feedback.

# Part 1: the goal, to be sent by the client

bool hasFrontWall

---

# Part 2: the result, to be sent by the server upon completion

duration time_elapsed

uint8 situationType

---

# Part 3: the feedback, to be sent periodically by the server during execution.

duration time_elapsed
